### PR TITLE
Updating mWater pull and merging with old field notes

### PR DESCRIPTION
**Pull request by SamStruthers**
In an effort to better track sensor malfunctions in livestreaming data, I added a sensor malfunction section to our field notes. I/techs/others can submit a survey if they see data that looks erroneous which may not be caught by QAQC (drift, broken sensor, etc). By sourcing 'grab_malfunction_record.R', you get `malfunction_records` which has start/end_dt, sensor affected and notes, this can then be incorporated into the function `kw_known_issues` with some kind of map function in the future. 

I also tried to match all the columns in `sensor_notes`and have looked at most data to make sure this was done correctly, have not yet run on entire script due to error with api pull (issue: #86 ). Once this is fixed, I can see if the field notes still work correctly downstream in pipeline. 

Review requested:
- Thoughts on how old and new field notes are merged and any possible introduced errors
- Folder structure of `mWater_pull_collate`, some scripts will not be used in QAQC (spreadsheet, files_missing and download_pics) but I will use them on regular basis. 
- Clarity/conciseness of `clean_mwater_notes.R` and`grab_sensor_notes.R` code

### CODE REVIEW

File: src/mWater_collate/grab_sensor_notes.R [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+         date = as.character(date)
+  )%>%
+  arrange(desc(DT_round))%>%
+  #order columns in easily readable ordering
+  select(site, crew, DT_round,sonde_employed,  sensors_cleaned, wiper_working, rdo_cap_condition, rdo_cap_replaced , ph_junction_replaced ,
+         cals_performed, cal_report_collected , sensor_malfunction,sensor_pulled,sensor_deployed, sensor_swapped_notes,
+         visit_type,start_time_mst,DT_join,  start_DT, end_dt,date,  visit_comments,photos_downloaded, field_season, last_site_visit)
+
+# back up to CSV
+write_csv(sensor_notes, "data/mWater_sensor_field_notes.csv")
```
Comment (juandlt-csu) [inline src/mWater_collate/grab_sensor_notes.R:36]: question: Is this the file that we are going to want to pull into our QAQC workflow?

File: src/mWater_collate/data_manager_tools.Rmd [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+-   Label according to site, date, description in the format site_YYYYMMDD_descriptor.jpg
+
+-   Only download photos which have not yet been downloaded
+
+```{r}
+source("src/mWater_collate/download_pictures.R")
+  
+#RUN TO DOWNLOAD NEW PICTURES
+# It takes about 2-5 minutes to download ~25-50 photos
+# Sometimes the request to mWater time out, just re run the function below if that happens
```
Comment (juandlt-csu) [inline src/mWater_collate/data_manager_tools.Rmd:61]: suggestion: you could do a try catch here that will catch a time out and rerun the function in that case.

File: src/mWater_collate/clean_mwater_notes.R [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+# correct columns where Other input is allowed (Site, visit type, photos downloaded, sensor malfunction)
+# Add rounded date time
+
+
+all_notes_cleaned <- read_csv(url(api_url),show_col_types = FALSE)%>%
+  mutate(
+    #start and end dt comes in as UTC -> to MST
+    start_DT = with_tz(ymd_hms(start_dt), tz = "MST"),
+    end_dt = with_tz(ymd_hms(end_dt), tz = "MST"),
+    malfunction_end_dt = with_tz(ymd_hms(malfunction_end_dt), tz = "MST"),
```
Comment (juandlt-csu) [inline src/mWater_collate/clean_mwater_notes.R:25]: question: where is malfunction_end_dt being created?

Comment (SamStruthers) [reply src/mWater_collate/clean_mwater_notes.R:25]: In Mwater, it is a user input

**Pull request merged**